dbscripts/xml/upgrade/3.2.1_preupdate_subeditor_categories.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE data SYSTEM "../../../lib/pkp/dtd/xmlData.dtd">
+
+<!--
+	* dbscripts/xml/upgrade/3.2.1_preupdate_subeditor_categories.xml
+	*
+	* Copyright (c) 2013-2020 Simon Fraser University
+	* Copyright (c) 2003-2020 John Willinsky
+	* Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+	*
+	* Rename section_editors table #5694
+	*
+	-->
+
+<data>
+	<sql>
+		<dropindex table="section_editors" index="section_editors_pkey" />
+		<dropindex table="section_editors" index="section_editors_context_id" />
+		<dropindex table="section_editors" index="section_editors_section_id" />
+		<dropindex table="section_editors" index="section_editors_user_id" />
```

Comment:
@asmecher do we need to recreate indexes for this table or does the migration toolset do that? Indexes in the schema here: https://github.com/pkp/pkp-lib/pull/5700/files#diff-9b18cc074810a08891edd65b157fdb53R289

Reply:
The ADODB tools will create the necessary indexes when the schema descriptor is synced on install (to create the tables) or upgrade (to adapt the tables to the current XML's state). (This will change when https://github.com/pkp/pkp-lib/issues/2493 is merged -- the schema files will not be synced on upgrade, and changes should be manually specified in `up()` and `down()` migration files. But that's later.)